pkg/backup/version.go

```diff
@@ -24,6 +24,7 @@ import (
 var compatibilityMap = map[string]map[string]struct{}{
 	"3.0": {"2.3": struct{}{}, "3.0": struct{}{}},
 	"3.1": {"3.0": struct{}{}, "3.1": struct{}{}},
+	"3.2": {"3.2": struct{}{}},
```

Comment:
Please just fix the upgrade guide.
Let's discuss separately about 3.2 support. Open a new issue?
Thanks!